Rules/CommonScripts/accolade_data.cfg

```
@@ -583,6 +592,7 @@ epsilon =
 	# participation 7 - OCE 1v1 knight tournament no. 5
 	# participation 8 - OCE 2v2 knight tournament no. 2
 	# participation 9 - OCE 1v1 archer tournament no. 2
+	# participation 10- OCE 2v2 mixed tournament no. 1
```

Missing space before hyphen.
```suggestion
	# participation 10 - OCE 2v2 mixed tournament no. 1
```